src/pageObjects/WelcomePage/WelcomePage.js

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable import/prefer-default-export */
+import BasePage from '../BasePage';
+import SignUpPopup from './components/SignUpPopup';
+
+export default class WelcomePage extends BasePage {
+  _welcomePageContentSelector = 'app-home';
+
+  _signUpBtnSelector = 'button:has-text("Sign up")';
+
+  constructor(page) {
+    super(page, '');
+    this.content = page.locator(this._welcomePageContentSelector);
+    this.signUpBtn = page.locator(this._signUpBtnSelector);
+  }
+
+  async navigate() {
+    await this._page.goto('');
+  }
```

Comment:
redundant method, you pas url is super()

method navigate will be available for each class which extends BasePage

Reply:
removed